feedsx/src/main/java/com/likeminds/feedsx/utils/ShareUtils.kt

```diff
@@ -3,12 +3,14 @@ package com.likeminds.feedsx.utils
 import android.content.Context
 import android.content.Intent
 import com.likeminds.feedsx.R
-import com.likeminds.feedsx.SDKApplication
 
 object ShareUtils {
     //share post with url using default sharing in Android OS
-    fun sharePost(context: Context, postId: String) {
-        val domain = (SDKApplication.getInstance()).getDomain()
+    fun sharePost(
+        context: Context,
+        postId: String,
+        domain: String
+    ) {
```

Comment:
Write proper documentation comment for this as this will be changed by customer dev this should have proper comment